### PR TITLE
CRD description updates

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

**Review by stuggi — APPROVED** [review]
/lgtm

File: api/v1beta1/baremetalset_types.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type BaremetalSetSpec struct {
 	// Replicas The number of baremetalhosts to attempt to aquire
 	// +kubebuilder:validation:Minimum=0
 	Replicas int `json:"replicas,omitempty"`
-	// Remote URL pointing to desired RHEL qcow2 image (TODO: currently gzipped only) for the set
+	// Remote URL pointing to desired RHEL qcow2 image (gzipped format is required)
```
Comment (abays) [inline api/v1beta1/baremetalset_types.go:31]: Gzipped format is actually not required.  We support compressed and uncompressed now

Comment (dprince) [reply api/v1beta1/baremetalset_types.go:31]: oh right. Lets drop it. Just trying to make this look better

Comment (dprince) [reply api/v1beta1/baremetalset_types.go]: @abays fixed in the PR below

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**